cryptographic_estimators/MQEstimator/MQAlgorithms/bjorklund.py

```diff
@@ -164,60 +151,53 @@ def _internal_memory_complexity_(_n, _m, _lambda):
         return log2(_internal_memory_complexity_(n, m, lambda_))
 
     def _compute_tilde_o_time_complexity(self, parameters: dict):
-        """
-        Return the Ō time complexity of Bjorklund et al.'s algorithm
-
-        TESTS::
+        """Return the Ō time complexity of Bjorklund et al.'s algorithm.
 
-            sage: from cryptographic_estimators.MQEstimator.MQAlgorithms.bjorklund import Bjorklund
-            sage: from cryptographic_estimators.MQEstimator.mq_problem import MQProblem
-            sage: E = Bjorklund(MQProblem(n=10, m=12, q=2), complexity_type=1)
-            sage: E.time_complexity(lambda_=7/10)
+        Tests:
+            >>> from cryptographic_estimators.MQEstimator.MQAlgorithms.bjorklund import Bjorklund
+            >>> from cryptographic_estimators.MQEstimator.mq_problem import MQProblem
+            >>> E = Bjorklund(MQProblem(n=10, m=12, q=2), complexity_type=1)
+            >>> E.time_complexity(lambda_=7/10)
             8.03225
         """
         n = self.nvariables_reduced()
         h = self._h
         return h + 0.803225 * n
 
     def _compute_tilde_o_memory_complexity(self, parameters: dict):
-        """
-        Return the Ō time complexity of Bjorklund et al.'s algorithm
-
-        TESTS::
-
-            sage: from cryptographic_estimators.MQEstimator.MQAlgorithms.bjorklund import Bjorklund
-            sage: from cryptographic_estimators.MQEstimator.mq_problem import MQProblem
-            sage: E = Bjorklund(MQProblem(n=10, m=12, q=2), complexity_type=1)
-            sage: E.memory_complexity(lambda_=7/10)
-            3
+        """Return the Ō time complexity of Bjorklund et al.'s algorithm.
+
+        Tests:
+            >>> from cryptographic_estimators.MQEstimator.MQAlgorithms.bjorklund import Bjorklund
+            >>> from cryptographic_estimators.MQEstimator.mq_problem import MQProblem
+            >>> E = Bjorklund(MQProblem(n=10, m=12, q=2), complexity_type=1)
+            >>> E.memory_complexity(lambda_=7/10)
+            3.0000000000000004
```

Comment:
could this cause problems depending on the machine / system the test is executed on? the result seems to be 3 and the ..00004 an issue of float impression. In case the impression is inconsistent between systems we might want to consider rounding the values first?

Reply:
I can't say for sure.  This was a temporal patch suggested by @Javierverbel, and I can't think in how to round it without introduce some undesirable pattern/artificial correction. If you have any idea, let me know and I could implement it.

Reply:
if we round to say 5 decimal digits, float precision should never be an issue., e.g., using 

`round(E.memory_complexity(lambda_=7/10),5)`

but probably we only want to do it if necessary, so maybe for now lets leave it like this and see if it causes any problems first